packages/scss/src/components/_button.scss

```diff
@@ -52,97 +52,89 @@
 		background: transparent;
 
 		&:hover, &:focus {
-			box-shadow: 0 0 0 2px $primary inset;
-			color: $primary;
+			box-shadow: 0 0 0 2px _get($luTheme, "palette.colors.primary") inset;
+			color: _get($luTheme, "palette.colors.primary");
 		}
 	}
 
 	&.mod-filling {
 		&:hover, &:focus {
-			background: $primary;
+			background: _get($luTheme, "palette.colors.primary");
 		}
 	}
 
 	/* Colors */
 
 	&.mod-primary {
-		background: $primary;
+		background: _get($luTheme, "palette.colors.primary");
 
 		&:active {
-			background: darken($primary, 5%);
+			background: darken(_get($luTheme, "palette.colors.primary"), 5%);
 		}
 	}
 
 	&.mod-secondary {
-		background: $secondary;
+		background: _get($luTheme, "palette.colors.secondary");
 		color: white;
 
 		&:active {
-			background: darken($secondary, 5%);
-		}
-	}
-
-	&.mod-lucca {
-		background: $lucca;
-
-		&:active {
-			background: darken($lucca, 5%);
+			background: darken(_get($luTheme, "palette.colors.secondary"), 5%);
 		}
 	}
 
 	&.mod-grey {
-		background: $grey;
+		background: _get($luTheme, "palette.colors.greyStandard");
 		color: white;
 
 		&:active {
-			background: darken($grey, 5%);
+			background: darken(_get($luTheme, "palette.colors.greyStandard"), 5%);
 		}
 	}
 
 	&.mod-grey-light {
-		background: $greyLighter;
+		background: _get($luTheme, "palette.colors.greyLight");
 
 		&:active {
-			background: darken($greyLighter, 5%);
+			background: darken(_get($luTheme, "palette.colors.greyLight"), 5%);
 		}
 	}
 
 	&.mod-error {
-		background: $red;
+		background: _get($luTheme, "palette.colors.error");
 		color: white;
```

Comment:
How can you be so sure the error color I'll set in my overrides isn't too pale for white text to be illegible over it?
In Lucca-ui, a palette color was a main color as well as the color text should have when writter over such a color.